### PR TITLE
Feature/new cell type fields

### PR DESCRIPTION
**Pull request by pinin4fjords**
A fairly simple PR, to increase flexibility in the cell type fields used. I've added the new cell type labels to the list of SDRF fields considered (though this could also have been done by providing the env var). Marker gene files are dynamic, related to fields by parsing their file names (though the appropriate cell group will have to have been added anyway for this to work).

### CODE REVIEW

**Comment by pinin4fjords** [comment]
Note: the CI issue is a perennial one to do with container setup in the CI we've not figured out yet. I confirm that the container-based tests pass locally.

File: bin/load_db_scxa_marker_genes.sh [hunk 1]
```diff
@@ -146,14 +144,19 @@ if [[ -z ${NUMBER_MGENES_FILES+x} || $NUMBER_MGENES_FILES -gt 0 ]]; then
...
 
-  if [ -e "$inferredCelltypeMarkers" ]; then
-    tail -n +2 $inferredCelltypeMarkers | awk -F'\t' -v EXP_ID="$EXP_ID" 'BEGIN { OFS = "|"; } { gsub("^nan$","Not available",$1); print EXP_ID"_inferred cell type_"$1, $4, $8 }' >> ${groupMarkerGenesToLoad}.tmp
-  fi
-  if [ -e "$authorsInferredCelltypeMarkers" ]; then
-    tail -n +2 $authorsInferredCelltypeMarkers | awk -F'\t' -v EXP_ID="$EXP_ID" 'BEGIN { OFS = "|"; } { print EXP_ID"_authors inferred cell type_"$1, $4, $8 }' >> ${groupMarkerGenesToLoad}.tmp  
-  fi
+  re='^[0-9]+$'
+    
+  for markerGenesFile in $(ls $EXPERIMENT_MGENES_PATH/${EXP_ID}.marker_genes*.tsv); do
```
Comment (suhaibMo) [inline bin/load_db_scxa_marker_genes.sh:152]: Here you are reading all the marker genes_*inferred_cell_type.tsv associated with authors and ontology labels?

**Comment by suhaibMo** [comment]
LGTM @pinin4fjords

**Comment by suhaibMo** [comment]
> Note: the CI issue is a perennial one to do with container setup in the CI we've not figured out yet. I confirm that the container-based tests pass locally.

Regarding the CI test. I leave that you guys.

**Review by suhaibMo — APPROVED** [review]

File: bin/load_db_scxa_marker_genes.sh (hunk 1 above)
Comment (pinin4fjords) [reply bin/load_db_scxa_marker_genes.sh:152]: Actually, anything matching the file pattern, just wanted it to be flexible.

**Comment by pinin4fjords** [comment]
Thanks @suhaibMo !

**Pull request merged**